src/viam/services/motion/client.py

```diff
@@ -131,6 +141,51 @@ async def move_on_globe(
         response: MoveOnGlobeResponse = await self.client.MoveOnGlobe(request, timeout=timeout)
         return response.success
 
+    async def move_on_globe_new(
+        self,
+        component_name: ResourceName,
+        destination: GeoPoint,
+        movement_sensor_name: ResourceName,
+        obstacles: Optional[Sequence[GeoObstacle]] = None,
+        heading: Optional[float] = None,
+        configuration: Optional[MotionConfiguration] = None,
+        *,
+        extra: Optional[Mapping[str, ValueTypes]] = None,
+        timeout: Optional[float] = None,
+    ) -> str:
+        """
+        **Experimental**: use move_on_globe instead.
+        Move a component to a specific latitude and longitude, using a ``MovementSensor`` to check the location.
+
+        Note: Is non blocking.
+
+        Args:
+            component_name (ResourceName): The component to move
+            destination (GeoPoint): The destination point
+            movement_sensor_name (ResourceName): The ``MovementSensor`` which will be used to check robot location
+            obstacles (Optional[Sequence[GeoObstacle]], optional): Obstacles to be considered for motion planning. Defaults to None.
+            heading (Optional[float], optional): Compass heading to achieve at the destination, in degrees [0-360). Defaults to None.
+            linear_meters_per_sec (Optional[float], optional): Linear velocity to target when moving. Defaults to None.
+            angular_deg_per_sec (Optional[float], optional): Angular velocity to target when turning. Defaults to None.
```

Comment:
What is the extra `optional` in `Optional[float], optional` for?

Reply:
I'm not sure, I was copying the pattern from `move_on_globe` which this is based on but which I didn't write. should this be just `Optional[float]`?

Reply:
updated